react/src/utils/api.ts

```diff
@@ -190,3 +198,18 @@ export const postAPIEmail = (email: string): Promise<boolean> => {
     .then(() => Promise.resolve(true))
     .catch((e) => Promise.reject(e));
 };
+
+export const postSubscriberDistrict = (
+  sub_id: string,
+  district: string
+): Promise<boolean> => {
+  const postData = querystring.stringify({
+    subscriber: sub_id,
+    district: district,
+    cid: uuid.callerID()
+  });
+  return axios
+    .post(Constants.UPDATE_DISTRICT_API_URL, postData, {})
+    .then(() => Promise.resolve(true))
+    .catch((e) => Promise.reject(e));
```

Comment:
[nit] these last 2 lines are confusing to me -- I guess I understand `Promise.resolve(true)` if it's important to have a boolean response, but unless Im super confused the catch is catching the error and immediately throwing it again, so it seems redundant

Reply:
mindlessly copypasta'd this, I'll reduce it to just the post